src/game-logic/reducers/purchaseForest.js

```diff
@@ -22,7 +22,7 @@ export const purchaseForest = (state, forestId) => {
   const { columns, price, rows } = PURCHASABLE_FOREST_SIZES.get(forestId)
 
   /*
-   * FIXME: using FOREST_AVAILABLE_NOTIFICATION here is temporary, this code path will
+   * TODO: using FOREST_AVAILABLE_NOTIFICATION here is temporary, this code path will
```

Comment:
FYI @lstebner I changed this from a `FIXME` to a `TODO`. I find it helpful to treat `FIXME`s as an indication that something that needs to be addressed before a PR is merged, whereas a `TODO` indicates an improvement to be optionally made at some point in the future.